back/src/Model/Group.ts

```diff
@@ -97,10 +126,12 @@ export class Group implements Movable {
         if (!this.currentZone) return;
 
         for (const user of this.positionNotifier.getAllUsersInSquareAroundZone(this.currentZone)) {
+            //  Todo: Merge two groups with a leader
             if (user.group || this.isFull()) return; //we ignore users that are already in a group.
             const distance = GameRoom.computeDistanceBetweenPositions(user.getPosition(), this.getPosition());
             if (distance < this.groupRadius) {
                 this.join(user);
+                this.setOutOfBounds(false);
```

Comment:
I'm not sure we should set the outOfBounds to false here.

The group might have a user that is outOfBounds and as long as this user is out of bounds, we should not set this to false.

Reply:
In case when a follower is to far away to the leader and the leader meet a new user the circle should appear.

Reply:
Mmmm... but the circle will disappear as soon as the follower moves, no? So the circle will only "blip" into sight and be hidden right after?

Reply:
No because line 267 we only hide the circle if there are only followers.